components/sidebar/Sidebar.styles.ts

```diff
@@ -0,0 +1,85 @@
+import styled from 'styled-components';
+
+import media from '../../modules/media';
+export const Menu = styled.ul`
+  ${(props) => props.hide && 'display: none;'}
+`;
+
+export const MenuItem1 = styled.li`
```

Comment:
You are using the same css multiple times. Obviously something is wrong here. Also, your styled-component is named MenuItem1, that means if I want to change the order of the menu items, I need to change the css as well. You need to always separate layout and business logic. Let's discuss this tomorrow.